src/backend/core/user/models.py

```diff
@@ -52,7 +52,7 @@ class User(AbstractBaseUser, PermissionsMixin):
     is_admin = models.BooleanField(default=False)
     is_staff = models.BooleanField(default=False)
 
-    cart = models.ForeignKey(Cart, null=True, on_delete=models.SET_NULL)
+    cart = models.OneToOneField(Cart, null=True, on_delete=models.SET_NULL)
```

Comment:
I think it would be better to set `ForeignKey` to user model on cart model. 
Since cart can/will store products, we will probably want to bind this cart to some order after checkout. 
With that in mind, we will want to store info in the cart (mainly products, etc.). However, having `cart` ref on `user` as OneToOne might complicate things a bit a not allow us to have some old cart stored in a database with ref to the user.

So I think might be better to have cart -> user as n:1

Reply:
If we want to bind our cart to order (which is a good idea IMO), then yes.